tests/testthat/test-connection.R

```diff
@@ -167,16 +156,15 @@ test_that("projects() can handle NULL owner in project JSON", {
   two_projects_json_no_owner_json <- TWO_PROJECTS_JSON
   two_projects_json_no_owner_json[[1]]$owner <- NULL
   m <- mock(two_projects_json_no_owner_json)
```

Comment:
what exactly is being mocked here? I can understand where you where using to count the number of times a function was being called but it seems here you are just trying to mock the output of a function in which case I think you can do it directly with local bindings?